lib/customer.rb

```diff
@@ -0,0 +1,37 @@
+require 'csv'
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
Do these need to be accessors?  Do users of the class need to be able to directly change email and address?

Default to making these readers instead of accessors.